spec/views/shared/_ready_for_the_day.html.erb_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe "shared/_ready_for_the_day.html.erb" do
+
+  it "contains Get Ready for the Day" do
+    render
+    response.should have_text("Get Ready for the Day")
```

Comment:
`.should` is the old (rspec 2.x) syntax (we now use rspec 3.x)

Update it so it is in this format:

`expect(response).to have_text("Get Ready for the Day")`